wolfssl/test.h

```diff
@@ -1681,6 +1687,24 @@ static WC_INLINE int myVerify(int preverify, WOLFSSL_X509_STORE_CTX* store)
                                                                   subject);
         XFREE(subject, 0, DYNAMIC_TYPE_OPENSSL);
         XFREE(issuer,  0, DYNAMIC_TYPE_OPENSSL);
+#if defined(SHOW_CERTS) && !defined(NO_FILESYSTEM)
+/* avoid printing same certs since myVerify is called for every cert in the chain */
```

Comment:
Please adjust this comment. The verify callback is called for every certificate only when `--enable-opensslextra` is defined because it sets `WOLFSSL_ALWAYS_VERIFY_CB` and `WOLFSSL_VERIFY_CB_ALL_CERTS`. Normal cases of the verify callback only occur on certificate failures when the  `wolfSSL_set_verify(ssl, SSL_VERIFY_PEER, myVerifyCb);` is called.